src/ensembl/stories/shared-components/accordion/Accordion.stories.scss

```diff
@@ -19,3 +19,30 @@
   height: 350px;
   overflow-y: auto;
 }
+
+.entityViewerAccordion {
+  border-color: transparent;
+}
+
+.entityViewerAccordionHeader {
+  background-color: $light-grey;
+}
+
+.entityViewerAccordionButton {
+  color: $blue;
+}
+
+.entityViewerAccordionItem {
+  border-color: transparent;
+}
+
+.entityViewerAccordionItem + .entityViewerAccordionItem {
+  border-color: transparent;
+}
```

Comment:
Isn't this the same as lines 35-37?

Reply:
Yes, but this is needed to override the default styles.

Reply:
But is the rule on 35-37 actually doing anything?

Reply:
No, it doesn't. I've removed it.